default_config.yaml

```diff
@@ -90,12 +90,19 @@ detection:
   threshold: 5.0
 
 cutting:
-  cutout_size: 20
+  cutout_size: 25
+
+measuring:
+  annulus_radii: [10, 15]
+  annulus_units: pixels
```

Comment:
There should be (and we should use) a aperture radius as well.  The default should be something like 1 FWHM -- meaning the units should be FWHM rather than pixels.  I'd then make the default annulus something like 10-12 FWHMs.  (That gives an area 44x the area of the aperture, so the nosie contribution should be small.)

Reply:
The apertures are determined by the ZP corrections. I think I remember that we discussed this and decided that we 100% want the apertures to match the ZP. I know this because I had that as a parameter and then ripped it out of the class when we settled on just copying over the aperture sizes from the ZP...

Reply:
Yeah OK.  We should have those zp apertures configured somewhere on the config file, though.

Reply:
They are pulled from the instrument. If you recall, I was complaining last week that there are too many of them...